<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cashier</class>
 <widget class="QDialog" name="Cashier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton
{background:rgb(120,170,220,111);font:2em;
border:1px solid rgb(121,189,143);
border-radius:5px;}

QPushButton:hover{
	background-color: rgb(255, 255, 255);
	
	border:2px solid rgb(121,189,143);
}
QLineEdit{
background:rgb(255,255,255);font:2em;
border:1px solid rgb(121,189,143);
border-radius:5px;
}
QDialog{
	background:rgb(232,241,252);
	
border:2px solid rgb(120,170,220);
}</string>
  </property>
  <widget class="QGraphicsView" name="UserPic">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>40</y>
     <width>121</width>
     <height>161</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:rgb(255,255,255);font:2em;
border:1px solid rgb(121,189,143);
border-radius:5px;
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="text">
    <string>Search</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="text">
    <string>Purchase</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Num:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Acc  :</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>210</y>
     <width>521</width>
     <height>201</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑 Light</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QTableView{background:rgb(255,255,255);font:2em;
border:2px solid rgb(121,189,143);
border-radius:5px;}
QHeaderView{
        border: none;
        border-bottom: 1px solid rgb(0, 78, 161);
        background: transparent;
       
}
QHeaderView::section:horizontal {
        border: 2px;
        color: rgb(2, 65, 132);
        background: transparent;
        padding-left: 5px;
}
QHeaderView::section:horizontal:hover {
        color: white;
        background: rgb(0, 78, 161);
}
QHeaderView::section:horizontal:pressed {
        color: white;
        background: rgb(6, 94, 187);
}
QHeaderView::up-arrow {
        width: 13px;
        height: 11px;
        padding-right: 5px;
       
        subcontrol-position: center right;
}

QHeaderView::down-arrow {
        width: 13px;
        height: 11px;
        padding-right: 5px;
       
        subcontrol-position: center right;
}

QTableView {
        border: 1px solid rgb(111, 156, 207);
        background: rgb(224, 238, 255);
        gridline-color: rgb(111, 156, 207);
}
QTableView::item {
        padding-left: 5px;
        padding-right: 5px;
        border: none;
        background: white;
        border-right: 1px solid rgb(111, 156, 207);
        border-bottom: 1px solid rgb(111, 156, 207);
}
QTableView::item:selected {
        background: rgba(0, 0, 0, 100);
}
QTableView::item:selected:!active {
        color: rgb(65, 65, 65);
}
QTableView::indicator {
        width: 20px;
        height: 20px;
}
</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>28</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton_5{
	border:2px solid rgb(120,170,220);
	;
	border-radius:6px;
background:url(E:/Qt code/BookSellSys/close.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>211</width>
     <height>116</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEdit">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QCheckBox" name="checkBox_2">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Author</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="checkBox_3">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Publisher</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="checkBox">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>Book Name</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Cashier：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>微软雅黑 Light</family>
       </font>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>UserPic</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>tableView</zorder>
  <zorder>pushButton_5</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_4</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>141</x>
     <y>139</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_5</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>264</x>
     <y>172</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_3</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>470</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>96</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit_2</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>473</x>
     <y>145</y>
    </hint>
    <hint type="destinationlabel">
     <x>517</x>
     <y>77</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>lineEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>484</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
